README.md

```diff
@@ -11,6 +11,88 @@ more cloud vendors:
 
 - Amazon Web Services (AWS): go to the [aws](aws) folder for more details.
 
+## Prerequisites
+
+1. Terraform installed
+2. AWS CLI set up with appropriate permissions
+3. A working knowledge of Terraform and AWS resources
+
+## Instructions
+
+### 1. Initialization
+
+Before applying any Terraform script, initialize your Terraform working directory:
+
+```bash
+$ cd aws/
+$ terraform init
```

Comment:
It's good idea to add `$` to indicate this is a shell script, but considering GitHub web has a convenient copy button in the code block, the `$` would be copied too so the users have to remove it after pasting it to their terminal, this seems to be an inconvenience. What do you think we remove the leading `$`s?